content/blog/monthly-july-2020.md

```diff
@@ -0,0 +1,80 @@
+---
+title: This Month in TiKV - July 2020
+date: 2020-07-30
+author: TiKV Authors
+---
+The warmest month in the Northern Hemisphere comes to an end soon. It’s time to wrap up everything we have achieved in this month!
+
+Let’s get started.
+
+## News
+
+TiKV is up for graduation from CNCF! This is a milestone that would not be possible without you! If you would like to support TiKV in the vote, follow the [instructions](https://lists.cncf.io/g/cncf-toc/join) to join TOC and post your replies [here](https://lists.cncf.io/g/cncf-toc/message/4902)!
```

Comment:
```suggestion
TiKV is up for graduation from CNCF! This is a major milestone we have been waiting for, which consummates the efforts and commitments of the TiKV community since it was born. It would not be possible without you, our contributors and adopters! If you would like to support TiKV in the vote, follow the [instructions](https://lists.cncf.io/g/cncf-toc/join) to join the CNCF TOC group and post your replies [here](https://lists.cncf.io/g/cncf-toc/message/4902)!
```